gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceSource.java

```diff
@@ -91,35 +75,23 @@ public class SalesforceSource extends QueryBasedSource<JsonArray, JsonElement> {
   public static final String USE_ALL_OBJECTS = "use.all.objects";
   public static final boolean DEFAULT_USE_ALL_OBJECTS = false;
 
-  private static final String ENABLE_DYNAMIC_PROBING = "salesforce.enableDynamicProbing";
-  private static final String DYNAMIC_PROBING_LIMIT = "salesforce.dynamicProbingLimit";
-  private static final int DEFAULT_DYNAMIC_PROBING_LIMIT = 1000;
-  private static final String MIN_TARGET_PARTITION_SIZE = "salesforce.minTargetPartitionSize";
-  private static final int DEFAULT_MIN_TARGET_PARTITION_SIZE = 250000;
-  // this is used to generate histogram buckets smaller than the target partition size to allow for more even
-  // packing of the generated partitions
-  private static final String PROBE_TARGET_RATIO = "salesforce.probeTargetRatio";
-  private static final double DEFAULT_PROBE_TARGET_RATIO = 0.60;
-  private static final int MIN_SPLIT_TIME_MILLIS = 1000;
-
-  private static final String DAY_PARTITION_QUERY_TEMPLATE =
-      "SELECT count(${column}) cnt, DAY_ONLY(${column}) time FROM ${table} " + "WHERE ${column} ${greater} ${start}"
-          + " AND ${column} ${less} ${end} GROUP BY DAY_ONLY(${column}) ORDER BY DAY_ONLY(${column})";
-  private static final String PROBE_PARTITION_QUERY_TEMPLATE = "SELECT count(${column}) cnt FROM ${table} "
-      + "WHERE ${column} ${greater} ${start} AND ${column} ${less} ${end}";
-
-  private static final String ENABLE_DYNAMIC_PARTITIONING = "salesforce.enableDynamicPartitioning";
-  private static final String EARLY_STOP_TOTAL_RECORDS_LIMIT = "salesforce.earlyStopTotalRecordsLimit";
+  @VisibleForTesting
+  static final String ENABLE_DYNAMIC_PROBING = "salesforce.enableDynamicProbing";
+  static final String MIN_TARGET_PARTITION_SIZE = "salesforce.minTargetPartitionSize";
```

Comment:
are these config props to be provided at job invocation?  if so, should be `public`